templates/default/config.json.erb

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Since you don't template anything here, can you move it to `files/default` and change the recipes above? For example look here https://github.com/logicalclocks/kagent-chef/blob/master/recipes/install.rb#L195 although that's for a whole directory

Reply:
Fixed